workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -27,22 +27,73 @@ public class LocatorConfig : ConnectionConfig
         /// </summary>
         public readonly LocatorParameters LocatorParameters = new LocatorParameters();
 
+        private const string UseOfMultipleCredentialTypesError = "SpatialOS login token and Steam credentials may not be set at the same time.";
+
         public LocatorConfig()
         {
             UseExternalIp = true;
         }
 
         /// <summary>
-        ///     Checks that the LocatorConfig instance is valid. This does not guarantee a successful connection.
+        ///    Checks that the connection configuration is valid. This does not guarantee a successful connection.
         /// </summary>
-        public override void Validate()
+        /// <param name="errorMessage">Reason for failing the validation in case false is returned.</param>
+        /// <returns>True, if the connection configuration is valid.</returns>
+        public override bool Validate(out string errorMessage)
         {
-            ValidateConfig(LocatorHost, RuntimeConfigNames.LocatorHost);
+            if (string.IsNullOrEmpty(LocatorHost))
+            {
+                errorMessage = string.Format(MissingConfigError, RuntimeConfigNames.LocatorHost);
+                return false;
+            }
```

Comment:
Multiple whitespace issues such as this one. Please lint the file

Reply:
Added missing newlines.